### PR TITLE
Mesh_3: Add an example (polyhedron inside a bouding polyhedron) in the User Manual

### LINKED ISSUE #1157: Mesh_3 plugin should be able to mesh surfaces with border  inside a closed polyhedral domain
Feature request: Load several meshes, potentially with borders) that are inside or that intersect a closed polyhedral domain, and create a tetrahedral mesh.

### PR DESCRIPTION
**Pull request by afabri**
## Summary of Changes

The example illustrates how to remesh the interior of a polyhedral domain with a surface (with or without boundary) inside the polyhedral domain. 

The constructor for this scenario existed already before.

Fix #1157.

## Release Management

* Affected package(s): Mesh_3

### CODE REVIEW

**Comment by lrineau** [comment]
The Travis job failed because of the permissions checking (in the `CHECK` line of Travis results):
```
.. Testing permissions...
Mesh_3/doc/Mesh_3/fig/polyhedral_domain_with_surface.jpg
Mesh_3/examples/Mesh_3/data/horizons-domain.off
Mesh_3/examples/Mesh_3/data/horizons.off
```
https://travis-ci.org/CGAL/cgal/jobs/237835904#L4063

**Comment by lrineau** [comment]
@afabri: The Travis job still fail. You have to change the permissions of the files and push the corresponding commit.

**Comment by lrineau** [comment]
(I will review the user manual, once it is merged in `integration`.)

**Comment by lrineau** [comment]
Sorry, I closed (and reopened) the PR by error.

**Comment by lrineau** [comment]
I fixed the permissions in https://github.com/CGAL/cgal/pull/2143/commits/b537432884318cdb281d6e9681345f4443b6cc8c.

File: Mesh_3/doc/Mesh_3/Mesh_3.txt [hunk 1]
```diff
@@ -654,13 +654,27 @@ Right : the output mesh with added 1D-features at the intersection of subdomains
...
-View of 3D meshes produced from a polyhedral domain. (i) is a view of file out_1.mesh and (ii) is a view of file out_2.mesh. Code from subsection \ref Mesh_3_subsection_examples_polyhedral generates these files."
+View of 3D meshes produced from a polyhedral domain. (i) is a view of file out_1.mesh and (ii) is a view of file out_2.mesh. Code from subsection \ref Mesh_3_subsection_examples_polyhedral generates these files.
+\cgalFigureEnd
+
+\subsubsection Mesh_33DSurfaceInsidePolyhedralDomain Remeshing a Polyhedral Domain with Surfaces
+
+In the following example we have a "bounding polyhedron" which defines the meshing domain and two surfaces inside this domain. 
+The surfaces inside the domain may be closed surfaces as well as surfaces with boundaries. In the case of a closed surface
+the volume delimited by this surface is also considered as inside the domain. That is different from the example in the 
+previous subsection.
```
Comment (lrineau) [inline Mesh_3/doc/Mesh_3/Mesh_3.txt]: > That is different from the example in the previous subsection.

It seems that sentence refers to a subsection that is not the previous one. The previous subsection is meshing a single polyhedron (the elephant).

**Comment by afabri** [comment]
I removed the sentence. In fact the documentation of the constructor is clear about that.

**Pull request merged**